assured-workloads-example/versions.tf

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = ">= 1.3"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.77"
+    }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 3.77"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = ">=3.6.2"
+    }
+  }
+}
```

Comment:
You should add a `provider_meta` here in order to make blueprint usage metrics available for Googlers:

```terraform
provider_meta "google" {
    module_name = "blueprints/terraform/kms-solutions:assured-workloads-example/v0.2.0"
}
provider_meta "google-beta" {
    module_name = "blueprints/terraform/kms-solutions:assured-workloads-example/v0.2.0"
}
```

Reply:
Added.